recipes/librerun-sdk/conda_build_config.yaml

```
@@ -0,0 +1,5 @@
+# Needed for Metal, the hardware acceleration api for osx
+MACOSX_SDK_VERSION:        # [osx]
+  - "11.0"                 # [osx]
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - '10.12'                # [osx]

```

This should only be set for osx-64
```suggestion
MACOSX_SDK_VERSION:        # [osx and x86_64]
  - "11.0"                 # [osx and x86_64]
MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
  - '10.12'                # [osx and x86_64]
```